orm/internal/codegen/query_go.go

```diff
@@ -0,0 +1,31 @@
+package codegen
+
+func (t tableGen) genQueries() {
+	if !t.genQueryServer {
+		return
+	}
+
+	name := t.msg.Desc.Name()
+
+	storeStructName := t.storeStructName()
+	// gen Get method
+	t.P("func (x ", storeStructName, "QueryServer) ", name, "(ctx ", contextPkg.Ident("Context"), ", req *", name, "Request) (*", name, "Response, error) {")
+	t.P("res, err := x.", storeStructName, ".", t.param(t.msg.GoIdent.GoName), ".Get(ctx, ", "")
+	for _, name := range t.primaryKeyFields.Names() {
+		for _, field := range t.msg.Fields {
+			if field.Desc.Name() == name {
+				t.P("req.", field.GoName, ",")
+			}
+		}
+	}
+	t.P(")")
+	t.P("if err != nil { return nil, err }")
+	t.P("return &", name, "Response{Value: res}, nil")
```

Comment:
Can this be specific? e.g. `Balances` rather than `Values`

Reply:
Sure